mithril-aggregator/src/configuration.rs

```diff
@@ -488,9 +488,8 @@ pub struct ServeCommandConfiguration {
     /// Signed entity types parameters (discriminants names in an ordered, case-sensitive, comma
     /// separated list).
     ///
-    /// The values `MithrilStakeDistribution` and `CardanoImmutableFilesFull` are prepended
-    /// automatically to the list.
-    #[example = "`MithrilStakeDistribution,CardanoImmutableFilesFull,CardanoStakeDistribution`"]
+    /// The value `MithrilStakeDistribution` is prepended is automatically to the list.
+    #[example = "`MithrilStakeDistribution,CardanoImmutableFilesFull,CardanoStakeDistribution,CardanoDatabase,CardanoTransactions`"]
```

Comment:
Maybe we can remove `MithrilStakeDistribution` from the example list, since it's added by default?
```suggestion
    #[example = "`CardanoImmutableFilesFull,CardanoStakeDistribution,CardanoDatabase,CardanoTransactions`"]
```